sct-commons/src/test/java/org/lfenergy/compas/sct/commons/ExtRefEditorServiceTest.java

```diff
@@ -928,4 +928,18 @@ void updateIedNameBasedOnLnode_when_no_matching_lnode_should_clear_binding(){
         assertExtRefIsNotBound(findExtRef(scd, "IED_NAME1", "LD_INST11", "STAT_LDSUIED_LPDO 1 Sortie_13_BOOLEAN_18_stVal_1"));
 
     }
+
+    @Test
+    void updateIedNameBasedOnLnode_when_several_ied_match_compasFlow_should_return_an_error() {
```

Comment:
In this test results there are also changes in `Extref` and  `compas:Flow`.   Could you be more specific?

Reply:
The result wanted is just the error in SclReportItem which makes the Scd unusable anyway.